docs/projects/centurion_erp/teams.md

```diff
@@ -0,0 +1,45 @@
+# Teams creation playbook
+
+The teams playbook has been created for the purpose of creating standardized teams with a defined set of permissions. With this playbook it is possible to create every team within an organization and define set teams for newly created team. The playbook can also be used to patch a team with updated permissions or notes.
+
+# Requirements
+
+The teams creation playbook has a number of requirements that MUST be satisfied in order to complete
+
+## Configuration
+
+The teams playbook uses variables that are located in a configuration inventory yaml file located within the host_vars folder
+
+### Structure
+
+The expected structure of the inventory file is:
+```yaml
+Centurion_erp:
+	common_permissions:
+	  name: $team-name "team name"
+	  permissions: $team-name-permissions []
+	  notes: $team-name-notes "any required notes"
+```
+
+ > the common_permissions uses anchors to set each team_name and permissions so that they can be specified once and used multiple times. This is useful if setting up multiple organizations that require standardized teams to be created.
+
+> If only one organization is being created the common_permissions is not required.
+
+```yaml
+	teams:
+    - name: "organization name"
+      teams:
+        - name: "team-name"
+          permissions: []
+          notes: "permissions must be a list"
+```	
+
+> If using yaml anchors replace the "team-name" with the anchor "$team-name"
+
+## AWX template
+The teams playbook includes the [AWX Feature](https://nofusscomputing.com/projects/ansible/playbooks/awx/) to import the playbook as a job template in AWX / Ansible Automation Platform.
+The following job template will be created:
+* **Centurion/access/teams** Creation and patching of teams and permissions
+
+On import to AWX / Ansible Automation Platform a credential type will also be created, `Playbook/teams/centurion` that can be used to supply the required secrets and Centurion host.
+
```

Comment:
Initial work on the teams documentation. Don't worry about the commit message, it will be changed / removed in the future. @jon-nfc feel free to make any suggestions on how this is looking so far